fhir-config/src/main/java/com/ibm/fhir/config/FHIRRequestContext.java

```diff
@@ -56,28 +56,29 @@ public FHIRRequestContext initialValue() {
             }
         }
     };
-    
+
     public FHIRRequestContext() {
         this.requestUniqueId = UUID.randomUUID().toString();
     }
-    
+
     public FHIRRequestContext(String tenantId) throws FHIRException {
+        this();
```

Comment:
only change here?

Reply:
yes

Reply:
and totally unrelated to the issue; forgot i had made this minor change and just didn't want to lose it